app/controllers/works_controller.rb

```diff
@@ -108,6 +108,8 @@ def resolve_ark
   # GET /works/1/edit
   # only non wizard mode
   def edit
+    @new_uploader = params[:new_uploader] == "true" || Rails.env.staging?
```

Comment:
Notice that with this PR the new file uploader is only available in staging (you can force it with a query string `?new_uploader=true` if you want to test locally)